common/test/fixture/IO/Map/mapobject.map

```diff
@@ -0,0 +1,418 @@
+// Game: Quake
```

Comment:
This is not used anywhere, right? It's just for manual testing?

Reply:
Yes. Maybe there is a better place we can put it

Reply:
I think it's ok to leave it there. I just wanted to make sure.